src/collection.ts

```diff
@@ -419,6 +423,27 @@ export class Collection<T extends TigrisCollectionType> extends ReadOnlyCollecti
 		});
 	}
 
+	/**
+	 * Deletes a single document in collection matching the filter
```

Comment:
please put a notice on what the behavior is if filter matches more than one doc.